assets/apps/customizer-controls/src/global-colors/PaletteColors.js

```diff
@@ -51,6 +58,10 @@ const PaletteColors = ({ values, defaults, save }) => {
 								return (
 									<ColorControl
 										disableGlobal
+										allowGradient={canUseGradient(
```

Comment:
this should be passed as a prop when defining the control for each individual color type.